pages/story.php

```diff
@@ -0,0 +1,19 @@
+<div class="container-fluid d-flex align-items-center min-vh-100" style="background-image: url('assets/images/bg-white.jpg')">
```

Comment:
```suggestion
<section class="container-fluid d-flex align-items-center min-vh-100" style="background-image: url('assets/images/bg-white.jpg')">
```
Remplace div by section